python/ql/src/semmle/python/regex.qll

```diff
@@ -472,8 +472,12 @@ abstract class RegexString extends Expr {
             this.getChar(endin) = "}" and
             end > start and
             exists(string multiples | multiples = this.getText().substring(start + 1, endin) |
+                multiples.regexpMatch("0+") and maybe_empty = true
```

Comment:
Is there a test for this? (I didn't spot one, but I might be overlooking something)

If there is an easy way to demonstrate we handle this properly now, that would be cool. If there isn't, it's not a major concern.

Reply:
Good spot, I added a simple test, thanks for pointing that out.